from opal.core.exceptions import Error


class SearchException(Error):
    pass
